client/src/app/interfaces/user.ts

```diff
@@ -0,0 +1,16 @@
+import { Stock } from './stock';
+import { Transaction } from './transaction';
+
+export interface User {
+    firstName: string;
+    lastName: string;
+    username: string;
+    email: string;
+    profilePicture: string;
```

Comment:
I do not know yet if backend model has a profilePicture field. If it does not, how would we get it? I actually think we may need to use a CDN for user pictures if we cannot store them in Neo4J (because maybe neo4j does not support picture storage. We could store the link of the picture from the CDN to neo4j).

Is the string in that case the url of the picture?

Reply:
yes this is something we will have to discuss. We have two options
1. Give the user a preset amount of images to chose from (easiest)
2. Let the user upload a picture of themselves and store it on something like amazon S3 (better but more time consuming)

Whats going to be stored in NEO4j is just the url string of the image, not the actual image. When we get closer to those stories, we'll see our progress and decide which option to choose